apps/ledger-live-desktop/src/renderer/families/stellar/AccountSubHeader.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import AccountSubHeader from "../../components/AccountSubHeader/index";
+
+export default function StellarAccountSubHeader() {
+  return (
+    <AccountSubHeader family="Stellar" team="Stellar Development Foundation"></AccountSubHeader>
```

Comment:
Don't we want to mention `Stellar Tokens` here as well?

Reply:
Indeed ! Done